wampproto/acceptor.py

```diff
@@ -73,7 +73,10 @@ def receive_message(self, msg: messages.Message) -> messages.Message:
 
                 return welcome
 
-            method = msg.authmethods[0]
+            method = Acceptor.ANONYMOUS
+            if msg.authmethods is not None and len(msg.authmethods) > 0:
+                method = msg.authmethods[0]
```

Comment:
```suggestion
                method = msg.authmethods[0]
```
use else to set `method = Acceptor.ANONYMOUS`